tests/SecurityCommandTest.php

```diff
@@ -2,46 +2,23 @@
 
 namespace Jorijn\LaravelSecurityChecker\Tests;
 
-use SensioLabs\Security\Result;
-use SensioLabs\Security\SecurityChecker;
-
 class SecurityCommandTest extends TestCase
 {
     public function testFireMethod()
     {
-        $resultMock = \Mockery::mock(Result::class);
-        $resultMock->shouldReceive('__toString')->andReturn(json_encode([]));
-
-        $securityCheckerMock = \Mockery::mock(SecurityChecker::class);
-        $securityCheckerMock->shouldReceive('check')->andReturn($resultMock);
-
-        // bind Mockery instance to the app container
-        $this->app->instance(SecurityChecker::class, $securityCheckerMock);
-
-        // call the command
-        $res = $this->artisan('security-check:now');
+        $this->setSafeBasePath();
 
-        // and check if it returns exit-code 0
-        $this->assertEquals($res, 0);
+        $this->assertEquals(0, $this->artisan(
+            'security-check:now'
+        ));
     }
 
     public function testFireMethodWithVulnerabilitiesFound()
     {
-        $resultMock = \Mockery::mock(Result::class);
-        $resultMock
-            ->shouldReceive('__toString')
-            ->andReturn(json_encode($this->getFakeVulnerabilityReport()));
-
-        $securityCheckerMock = \Mockery::mock(SecurityChecker::class);
-        $securityCheckerMock->shouldReceive('check')->andReturn($resultMock);
-
-        // bind Mockery instance to the app container
-        $this->app->instance(SecurityChecker::class, $securityCheckerMock);
-
-        // call the command
-        $res = $this->artisan('security-check:now');
+        $this->setVulnerableBasePath();
 
-        // and check if it returns exit-code 1 (error state)
-        $this->assertEquals($res, 1);
+        $this->assertEquals(1, $this->artisan(
```

Comment:
There's a problem with this approach: It actually queries the vulnerability database. I don't like it that an external service can influence whether my tests succeed or not. What happens when a dependency listed as `safe` does get affected somehow? I think it's better to mock the response from `analyzeDependencies` (etc.) so we get a predictable outcome.

Reply:
Sure thing, I'll convert the integration test into a unit test. That would take care of the latter 2 comments. 👍